### ISSUE TITLE
Post-mortem debugging in `excepthook` no longer works.

### ISSUE THREAD

**Issue by inducer** [inducer/pudb#552]
See discussion in #549. PRs welcome.

cc @presto-dev

**Comment by indigoviolet** [comment]
Is this still happening? Trying to figure out if post-mortem debugging is expected to work or not.

**Comment by inducer** [comment]
It is if you submit a PR? :slightly_smiling_face:

**Comment by indigoviolet** [comment]
@inducer Makes sense, thank you for confirming. I'm happy to try, but I don't know enough about why https://github.com/inducer/pudb/commit/f77332c5586dccb96cd85b9b92477ea6f749dd65 was made in the first place. Do you have any pointers on what the correct behavior should be, or what should be tested to make sure I don't break something else? Or should I just revert that commit?

**Comment by inducer** [comment]
TBH, I don't remember either. If you can make it work based on your testing, I think that's probably a step in the right direction. It's not a feature I use, so I'm not surprised it bitrotted.

**Comment by jfmoulin** [comment]
I have been missing the post-mortem debugging  for a while, really hope to see it back!
